src/audio/volume/volume_ipc4.c

```diff
@@ -156,6 +156,7 @@ int volume_init(struct processing_module *mod)
 	}
 
 	md->private = cd;
+	cd->is_same_volume = true;
```

Comment:
This initialize is different vs. proposed for ipc3. Also I wonder if false, 0 would be better default here.